login-registration/src/App.vue

```diff
@@ -0,0 +1,82 @@
+<template>
```

Comment:
I think this app is not running try to write proper code so that you can understand all concepts of vue and dev processes.